pkg/kapis/notification/v2beta2/handler.go

```diff
@@ -0,0 +1,100 @@
+package v2beta2
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"kubesphere.io/api/notification/v2beta2"
+
+	"github.com/emicklei/go-restful"
+)
+
+const (
+	VerifyUrl = "http://notification-manager-svc.kubesphere-monitoring-system.svc:19093/api/v2/verify"
```

Comment:
Use a configurable variable instead is more appropriate. I think the host is not always `http://notification-manager-svc.kubesphere-monitoring-system.svc:19093`.